config/example_config.yaml

```
@@ -26,3 +26,7 @@ languages:
   - language: go
     buildImage: golang:1.14
     runImage: docker.pkg.github.com/grpc/test-infra/go
+
+  - language: python
+    buildImage: l.gcr.io/google/bazel:latest
+    runImage: gcr.io/grpc-testing/e2etest/runtime/python3.7
```

Let's not reference a specific path and registry, since it is an example:

```suggestion
    runImage: docker.pkg.github.com/grpc/test-infra/python
```

---

Got it :). Thank you.